### PR TITLE
Remove feature flags

### PR DESCRIPTION
**Pull request by ashagross**
[25828](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25828)

## Description
Remove 526 Feature flags that are no longer needed.

## Testing done
visual/manual

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — COMMENTED** [review]
I think [this page](https://github.com/department-of-veterans-affairs/vets-website/blob/3e790c92fd8bc94df7562155fea045d8d1a7d330/src/applications/static-pages/wizard/index.js#L7) uses the 526 selector that is being removed here? Not sure though.

**Comment by Mottie** [comment]
Yeah we need to remove the flag from the wizard. Nice catch Micah!

**Review by mdewey — APPROVED** [review]
LGTM! Thanks for the clean up :-)

**Review by jbalboni — APPROVED** [review]

**Review by U-DON — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Review by dat-gcio — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 1]
```diff
@@ -205,13 +203,6 @@ const formConfig = {
           uiSchema: militaryHistory.uiSchema,
           schema: militaryHistory.schema,
           onContinue: captureEvents.militaryHistory,
-          appStateSelector: state => ({
-            dob: state.user.profile.dob,
-            allowBDD:
-              form526BDDFeature(state) && state.form.data?.['view:isBddData'],
-            servicePeriods:
-              state.form.data?.serviceInformation?.servicePeriods || [],
-          }),
```
Comment (Mottie) [inline src/applications/disability-benefits/all-claims/config/form.js:214]: we still need `dob` & `servicePeriods`

**Review by Mottie — CHANGES_REQUESTED** [review]

File: src/applications/disability-benefits/all-claims/config/form.js (hunk 1 above)
Comment (ashagross) [reply src/applications/disability-benefits/all-claims/config/form.js:214]: Got that added back!

File: src/applications/static-pages/wizard/index.js [hunk 2]
```diff
@@ -193,8 +192,4 @@ Wizard.defaultProps = {
   setReferredBenefit: () => {},
 };
 
-const mapStateToProps = state => ({
-  allowBDD: form526BDDFeature(state),
-});
-
-export default connect(mapStateToProps)(Wizard);
+export default connect(Wizard);
```
Comment (Mottie) [inline src/applications/static-pages/wizard/index.js:195]: We can remove the `connect`.

**Review by Mottie — APPROVED** [review]
ghe-emoji

**Pull request merged**